### PR TITLE
Dashboard updates

### PR DESCRIPTION
**Pull request by leifmadsen**
Update dashboards using registry.redhat.io/rhel8/grafana:7 and remove
any events or alarms from the panels.

### CODE REVIEW

**Comment by leifmadsen** [comment]
I have a bit more work to do here, but opening this up for a quick conversation to make sure I don't spend too much time doing something dumb :)

Two main bits here:

* remove reference to event stuff
* export this from RHEL grafana 7 to try and get us more aligned to a supported workload artifact

As part of the re-export (which seems to have not been done in a really long time), one of the plugins (which provides the hex pattern layout of the virtual machines) is updated from a very old version. I was having some issues with it not loading, which seems to have been resolved by the version bump. Might not have been related issues, but getting to something more recent is probably a good thing.

I will re-test all of this to make sure it works with the default image as well (from docker in the grafana org), but the changes look to be so minor that a newer grafana should be able to support this as well.

**Comment by csibbitt** [comment]
I was initially confused by your comment about "a very old version" since I see all the version numbers going down from "8.0.6" to "7.5.15", but then I looked closer and found the "6.5.1" references. I thought this is ironic because IIRC, Perses is closer to compatibility with Grafana 6. in general this seems to be moving in the right direction, though, doesn't seem dumb.

File: deploy/stf-1/rhos-cloud-dashboard.yaml [hunk 1]
```diff
@@ -7,9 +7,9 @@ metadata:
   namespace: service-telemetry
 spec:
   name: rhos-cloud-dashboard.json
-  plugins: 
+  plugins:
     - name: grafana-polystat-panel
-      version: "1.2.2"
+      version: "1.2.11"
```
Comment (leifmadsen) [inline deploy/stf-1/rhos-cloud-dashboard.yaml:12]: This _must_ stay at 1.2.x because 2.x series requires Grafana 8 or later.

**Comment by leifmadsen** [comment]
image

image

image

image

```yaml
oc get stf default -oyaml
apiVersion: infra.watch/v1beta1
kind: ServiceTelemetry
metadata:
  creationTimestamp: "2023-10-24T14:56:28Z"
  generation: 3
  name: default
  namespace: service-telemetry
  resourceVersion: "81961037"
  uid: 81945638-cd86-495b-ae69-585a2786a78f
spec:
  alerting:
    alertmanager:
      storage:
        persistent:
          pvcStorageRequest: 20G
        strategy: persistent
    enabled: true
  backends:
    metrics:
      prometheus:
        enabled: true
        scrapeInterval: 10s
        storage:
          persistent:
            pvcStorageRequest: 20G
          retention: 24h
          strategy: persistent
  clouds:
  - metrics:
      collectors:
      - bridge:
          ringBufferCount: 15000
          ringBufferSize: 16384
          verbose: false
        collectorType: collectd
        debugEnabled: false
        subscriptionAddress: collectd/cloud1-telemetry
      - bridge:
          ringBufferCount: 15000
          ringBufferSize: 16384
          verbose: false
        collectorType: ceilometer
        debugEnabled: false
        subscriptionAddress: anycast/ceilometer/cloud1-metering.sample
      - bridge:
          ringBufferCount: 15000
          ringBufferSize: 65535
          verbose: false
        collectorType: sensubility
        debugEnabled: false
        subscriptionAddress: sensubility/cloud1-telemetry
    name: cloud1
  graphing:
    enabled: true
    grafana:
      baseImage: registry.redhat.io/rhel8/grafana:7
      ingressEnabled: true
  observabilityStrategy: use_redhat
  transports:
    qdr:
      auth: none
      certificates:
        caCertDuration: 70080h
        endpointCertDuration: 70080h
      enabled: true
      web:
        enabled: false
```

```bash
oc get pods
NAME                                                        READY   STATUS    RESTARTS        AGE
alertmanager-default-0                                      3/3     Running   0               28h
cert-manager-operator-controller-manager-54f7679467-46pjj   2/2     Running   0               29h
default-cloud1-ceil-meter-smartgateway-7dfb95fcb6-bs6jl     3/3     Running   2 (4h22m ago)   28h
default-cloud1-coll-meter-smartgateway-674d88d8fc-858jk     3/3     Running   2 (4h22m ago)   28h
default-cloud1-sens-meter-smartgateway-9b869695d-xcssf      3/3     Running   1 (4h22m ago)   28h
default-interconnect-7cc45b86c5-lpljv                       1/1     Running   0               4h22m
grafana-deployment-54c54fddd7-grvx2                         2/2     Running   0               4h3m
grafana-operator-controller-manager-854d84f945-qz5tw        2/2     Running   0               6h27m
interconnect-operator-7bb99c5ff4-l6xc2                      1/1     Running   0               29h
prometheus-default-0                                        3/3     Running   0               28h
service-telemetry-operator-687b977879-ntfcn                 1/1     Running   0               20h
smart-gateway-operator-7d557cb7b7-9ppls                     1/1     Running   0               29h
```

**Pull request merged**